pytorch_lightning/trainer/trainer.py

```diff
@@ -1852,7 +1851,7 @@ def slurm_job_id(self) -> Optional[int]:
     @property
     def lightning_optimizers(self) -> List[LightningOptimizer]:
         if self._lightning_optimizers is None:
-            self.convert_to_lightning_optimizers()
+            self._convert_to_lightning_optimizers()
```

Comment:
This returns a list but this call converts to dict?

Reply:
Weird. Changed the return type to dict